apps/Makefile

```diff
@@ -109,6 +113,7 @@ ip65.atr: com
 	cp dos.sys      atr/dos.sys
 	cp dup.sys      atr/dup.sys
 	cp date.com     atr/date.com
+	cp datex.com    atr/datex.com
```

Comment:
So far the idea was always that there are separate disk images for separate cc65 targets. I'm wondering why we break up here with that tradition.

Based on what we did all those past years there would be something like a `date.comxl` that would go as `date.com` into something like `ip65.atrxl`.

Just think about some upcoming Wiki page describing an app. That page would write something about 'date' but not about 'datex'!

Reply:
> So far the idea was always that there are separate disk images for separate cc65 targets. I'm wondering why we break up here with that tradition.

I was not aware of such a tradition. I only remember Contiki where at some point in time there were both `atari` and `atarixl` versions. But then I only considered the space restriction on the disk to be the reason to have two ATRs.

> Based on what we did all those past years there would be something like a `date.comxl` that would go as `date.com` into something like `ip65.atrxl`.

I don't remember any *.comxl nor *.atrxl files. Can you help my memory?

> Just think about some upcoming Wiki page describing an app. That page would write something about 'date' but not about 'datex'!

Yes, `datex.com` is probably a bad name. `datexl.com` would be better IMO.